src/base/rss/rssarticle.h

```diff
@@ -49,18 +49,18 @@ namespace Rss
         Q_OBJECT
 
     public:
-        Article(Feed *parent, const QString &guid);
+        Article(Feed *parent);
 
         // Accessors
         bool hasAttachment() const;
-        const QString &guid() const;
+        const QString guid() const;
```

Comment:
no need to specify `const` for returned QString (also applies to functions below)